v2/pkg/astnormalization/directive_include_skip_test.go

```diff
@@ -54,4 +54,178 @@ func TestDirectiveIncludeVisitor(t *testing.T) {
 					skip: dog {__typename}
 				}`)
 	})
+	t.Run("include variables true", func(t *testing.T) {
+		runWithVariables(t, directiveIncludeSkip, testDefinition, `
+				query($yes: Boolean!) {
+					dog {
+						... @include(if: $yes) {
+							includeName: name
+						}
+					}
+					withAlias: dog {
+						name @include(if: $yes)
+					}
+				}`, `
+				query($yes: Boolean!) {
+					dog {
+						... {
```

Comment:
@devsergiy do we have another rule that inlines fragments/inline fragments with no conditions?

Reply:
yeah, it is fragment spread inlining modified in this PR

in order to apply it here

pass inline fragment as the main one and skip include as additional